meteor/testing.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+
+def assert_phases_allclose(array1: np.ndarray, array2: np.ndarray, atol=1e-3):
```

Comment:
You probably have a good reason for this, but why have this in testing instead of utils?